### PR TITLE
Course Dashboard screen automation code

### PR DESCRIPTION
**Pull request by bilalawan321**
### Description
- [LEARNER-7962](https://openedx.atlassian.net/browse/LEARNER-7962)

Following features have been implemented:
Verify that Course Dashboard screen will show following contents:
Header contents
Back icon  
Specific "<course name>" as Title 
Share icon
Verify that user should be able to go back by clicking Back icon
Verify that user should be able to view these Course contents:
Course Image 
Course Name
Course Provider
Course Ending date 
Last accessed(if any) 
Course Content
Verify on tapping "Videos" tab will load Videos screen
Verify on tapping "Discussion" tab will load Discussions screen
Verify on tapping "Dates" tab will load Dates screen
Verify on tapping "Resources" tab will load Resources screen

### CODE REVIEW

**Comment by bilalawan321** [comment]
@farhan-arshad-dev  can you please review this PR. ?

File: tests/android/tests/test_android_course_dashboard.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+            setup_logging.info('No course enrolled by this user.')
+
+        assert android_course_dashboard_page.get_navigation_icon().get_attribute('content-desc') \
+            == strings.COURSE_DASHBOARD_NAVIGATION_ICON
+        android_course_dashboard_page.get_navigation_icon().click()
+        assert android_main_dashboard_page.on_screen() == global_contents.MAIN_DASHBOARD_ACTIVITY_NAME
+        android_my_courses_list_page.load_course_details_screen()
+
+        if course_name:
+            assert android_course_dashboard_page.get_all_text_views()[0].text in course_name
```
Comment (farhan-arshad-dev) [inline tests/android/tests/test_android_course_dashboard.py:69]: I think this check is the same as [Line# 74](https://github.com/edx/edx-app-test/pull/63/files#diff-f2dfa6c077917093a5c018d76616404dfb38a140608b298ebb36a27e21379ee6R74) both are checking the title of the page.

**Review by farhan-arshad-dev — CHANGES_REQUESTED** [review]

File: tests/common/strings.py [hunk 2]
```diff
@@ -64,7 +64,7 @@
 LOGIN_PRIVACY = 'Privacy Policy'
 
 # WHATS NEW SCREEN
-WHATS_NEW_DONE = 'Done'
+WHATS_NEW_DONE = 'DONE'
```
[the remark this answers is not in the document]
Comment (bilalawan321) [reply tests/common/strings.py]: fixed

File: tests/android/tests/test_android_course_dashboard.py (hunk 1 above)
Comment (bilalawan321) [reply tests/android/tests/test_android_course_dashboard.py:69]: @farhan-arshad-dev, line 69 checks the header title of page, while line 74 checks the "course name" which is shown on course image, i have updated function name for clarity.

**Comment by bilalawan321** [comment]
@farhan-arshad-dev  can you plz check now, ?

File: tests/android/tests/test_android_course_dashboard.py (hunk 1 above)
Comment (farhan-arshad-dev) [reply tests/android/tests/test_android_course_dashboard.py:69]: better to add doc string for Line# 74

Comment (farhan-arshad-dev) [reply tests/android/tests/test_android_course_dashboard.py:69]: Also one is screen title and the second is course name/title

**Review by farhan-arshad-dev — CHANGES_REQUESTED** [review]

**Review by farhan-arshad-dev — APPROVED** [review]

**Pull request merged**